loop_rate:        1.5    # The frequency in Hz at which to run the control loops.
queue_size:       10     # The size of the message queue used for publishing and subscribing to topics.
resolution:       1.0    # The grid map underlying the area division will be downsampled to this resolution in meter / cell.
swarm_timeout:    5.0    # The time in seconds communication in the swarm can be delayed at most. Used to wait after an area division event before starting the area division or time after which it is assumed that a swarm member has left the swarm if no position update has been received.
visualize:        true   # Whether to publish the area division on a topic for visualization.
states:                  # Only CPSs in these states divide the area among each other.
- Coverage
- Tracking
- SelectRover
- LocalCoverage
optimizer:
  iterations:     100    # Maximum number of iterations of the optimization algorithm.
  variate_weight: 0.01   # Maximum variate weight of connected components.
  discrepancy:    30     # Maximum difference between number of assigned grid cells to each CPS.
